README.md

```diff
@@ -55,7 +55,11 @@ The script to run before generating the filediff comment.
 
 **`file_details_open` (optional, default: false)**
 
-Open the file details when the comment is created
+Open the file details when the comment is created.
+
+**`remove_filediff_comment` (optional, default: false)**
+
+Removes previous filediff comments before creating a new one.
```

Comment:
I would prefer the default to be true. Mainly because in a PR with lots of commits the comment will become frustrating.

Reply:
Good call 👍 I was only considering my upcoming use case (e.g. triggering the action on a `/filediff` comment). However, I could see other configurations (e.g. triggering on every commit to a PR) being very noisy.

Updated: https://github.com/Shopify/filediff/pull/1/commits/8e84fa92a51e9bc880d7e5390c93469f2df1c39d